examples/transfer.js

```diff
@@ -0,0 +1,67 @@
+// Import the necessary classes from the '@polkadot/api' and '@polkadot/keyring' modules
+const { ApiPromise } = require('@polkadot/api');
+const { Keyring } = require('@polkadot/keyring');
+
+async function main () {
+  // Log the start of the execution
+  console.log("Starting script...");
+
+  // Instantiate the API
+  console.log("Creating API promise...");
+  const api = await ApiPromise.create();
+
+  // Construct the keyring after the API (crypto has an async init)
+  console.log("Creating keyring...");
+  const keyring = new Keyring({ type: 'ethereum' });
```

Comment:
Okay, this is very interesting. I'm guessing that when I use Apps, it is not setting the ethereum keyring type correctly.

Thank you for this script.